src/native/libs/System.Native/pal_uid.c

```diff
@@ -237,3 +238,77 @@ int32_t SystemNative_GetGroups(int32_t ngroups, uint32_t* groups)
 
     return getgroups(ngroups, groups);
 }
+
+char* SystemNative_GetUName(uint32_t uid)
```

Comment:
(nit) It might be easier to read if you fully named the methods:

SystemNative_GetUserName
SystemNative_GetGroupName